modules/api/src/main/java/org/apache/ignite/schema/definition/builder/PrimaryKeyDefinitionBuilder.java

```diff
@@ -31,6 +32,14 @@
      */
     PrimaryKeyDefinitionBuilder withAffinityColumns(String... cols);
 
+    /**
+     * Sets affinity columns.
+     *
+     * @param cols Affinity columns. Must be a valid subset of ley columns.
+     * @return Primary index builder.
+     */
+    PrimaryKeyDefinitionBuilder withAffinityColumns(Set<String> cols);
```

Comment:
Does column order matter?
Are we going to build an index on primary key implicitly ?